File: 프로그래머스/lv1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
    let result = [];

    // 순회하며 조건이 참이면 배열에 넣음
    arr.forEach((item) => {
        if (item % divisor === 0) result.push(item);
    });

    // 배열에 아무요소도 없으면 -1 넣음
    if (result.length === 0) result.push(-1);

    // 정렬해서 반환
    return result.sort( (a, b) => a - b);
}

File: 프로그래머스/lv1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] arr, int divisor) {
        int[] result = Arrays.stream(arr)
                .filter(item -> item % divisor == 0) // 참인 요소들로 이루어진 배열 반환
                .sorted()   // 정렬 default: 이름사전순
                .toArray(); // 위에서 반환된 Stream객체 -> 배열
        if(result.length == 0) return new int[]{-1};
        return result;
    }
}